### ISSUE TITLE
SC2016 false positive for bash's PS0 ("Expressions don't expand in single quotes")

### ISSUE THREAD

**Issue by ilyagr** [koalaman/shellcheck#2156]
Online at https://www.shellcheck.net/, the following script triggers rule SC2016 ("Expressions don't expand in single quotes")

```
#!/bin/bash
PS0='`history -a && history -c && history -r`'$PS0
```
Result:
```
$ shellcheck myscript
 
Line 2:
PS0='`history -a && history -c && history -r`'$PS0
    ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.
```

I believe that's a false positive. PS0 is a new bash feature. Replacing PS0 by PS1 removes the warning ("No issues detected!").
